examples/vanilla/index.html

```diff
@@ -221,6 +221,7 @@ <h5>Remote Video</h5>
           login: document.getElementById('username').value,
           password: document.getElementById('password').value,
           ringtoneFile: './sounds/incoming_call.mp3',
+          // ringbackFile: './sounds/ringback_tone.mp3',
```

Comment:
Did you mean to leave in this comment?

Reply:
Yes, it is only to show that exists the option.